modules/schema/src/test/java/org/apache/ignite/internal/schema/BinaryTupleComparatorTest.java

```diff
@@ -348,8 +352,16 @@ public void partialComparatorAsciiTest() {
         assertTrue(partialBinaryTupleMatcher.match(tuple3, tupleReference) > 0);
         assertEquals(0, partialBinaryTupleMatcher.match(tuple1.limit(8).slice().order(ByteOrder.LITTLE_ENDIAN), tupleReference));
         assertTrue(partialBinaryTupleMatcher.match(tuple2.limit(8).slice().order(ByteOrder.LITTLE_ENDIAN), tupleReference) > 0);
+        assertTrue(partialBinaryTupleMatcher.match(
+                tuple2.limit(8).slice().order(ByteOrder.LITTLE_ENDIAN),
```

Comment:
Where do these limit values come from?

Reply:
From my mind.
It is an imitation of truncated value in B-tree entry inline.

Reply:
Don't we have a constant or configuration value for that?